### ISSUE TITLE
Circles creation only for certain users/groups

### ISSUE THREAD

**Issue by unclesam87** [nextcloud/circles#181]
Hey, i luv ur app.

I would luv an function to limited the creation of circles to certain users or groups, so that normal Users can still join circles but the creation of new circles would be limited to this users/groups.

Thank you very much:)

**Comment by amzen** [comment]
hello, this feature is standard.  In /settings/apps you can limit Circles to a special Group

**Comment by unclesam87** [comment]
Yeah but this locks the app for this group in total, but I would like people still be able to join a group or accept an invention

**Comment by breitero** [comment]
Hi, we serve a nextcloud platform for our pupils at school. Just to prevent that every kid creates its own 8 to 20 circles we want to restrict creating circles only to the teacher group. I know that this kind of feature was applied, actually it's not possible.
Nextcloud 20.0.4 stable, circles 0.20.6
Why disappeared that feature, I know it was there once...

**Comment by datapronu** [comment]
> This is in the code. So this ticket can be closed occ config:app:set circles limit_circle_creation --value

I cannot find this in documentation. How do I use this setting? --value is a comma separated list of groups and / or users?
